contents/blog/posthog-vs-fathom.mdx

```diff
@@ -0,0 +1,178 @@
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+PostHog and Fathom Analtyics both provide alternatives to Google Analytics. In fact, both explicitly mention it with marketing copy like "Switch to a better Google Analytics alternative."
+
+Both products feature web analytics dashboards with an emphasis on simplicity and privacy. They're ideal choices for customers that want to analyze and understand web traffic but find Google Analytics to be needlessly complicated.
+
+But what makes them different?
+
+## How is PostHog different?
+
+### 1. PostHog offers web analytics, and a lot more
+
+[Web analytics](/web-analytics) is a core feature of PostHog, but we offer many more tools to help you analyze, test, and observe your site and your product.
+
+These include [product analytics](/product-analytics), [session replay](/session-replay), [surveys](/surveys), [experiments](/experiments), [feature flags](/feature-flags), a [data warehouse](/data-warehouse), and more.
+
+### 2. PostHog has a generous free tier
+
+PostHog's ["ridiculously cheap" tier](/pricing) includes all of the features of PostHog for free with an allowance of 1M events per month. For Fathom, the pricing for up to 1M data points is $60 per month or 100K data points for $15 per month.
+
+Beyond PostHog's free tier, we try to be as cheap as possible and [transparent](/docs/billing/estimating-usage-costs). Want to know how much we'll charge? See our [pricing calculator](/pricing#calculator).
+
+PostHog also does not require a credit card to sign up and try the free tier.
+
+### Is Fathom simpler than PostHog?
+
+Although PostHog has more features than Fathom, it isn't necessarily more complicated.
+
+If you look at the web analytics dashboard for both products, they're pretty similar in appearance and the data provided:
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_dashboard_light_bc76a94566.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_dashboard_dark_42ac39a5f7.png"
+  classes="rounded"
+  alt="PostHog web analytics dashboard"
+/>
+
+![Fathom Analytics dashboard](https://res.cloudinary.com/dmukukwp6/image/upload/fathom_analytics_dashboard_195049b289.png)
+
+## Web analytics
+
+Both Fathom and PostHog are [Google Analytics alternatives](/blog/ga4-alternatives), so they're ideal for tracking and analyzing your website.
+
+<ComparisonTable column1="PostHog" column2="Fathom">
+  <ComparisonRow column1={true} column2={true} feature="Pageviews" description="Track visitors and their views" />
+  <ComparisonRow column1={true} column2={true} feature="Sessions" description="Track unique sessions and their durations" />
```

Comment:
Looking again, does Fathom have sessions? They have visitors, but it seems if a visitor visits more than once it doesn't count as a new session. 

![CleanShot 2024-11-08 at 14 21 48@2x](https://github.com/user-attachments/assets/37a39c95-ee6b-41b0-882d-6c577b79e7f4)